src/spikeinterface/extractors/shybridextractors.py

```diff
@@ -2,7 +2,7 @@
 
 import numpy as np
 
-from probeinterface import read_prb, write_prb
+import probeinterface as pi
```

Comment:
I am not really sure that "pi" is a good alias!!!!
why not keeping flat import ?

Reply:
is the same as `import spikeinterface as si`

Reply:
plus it's like that in many other files..let's unify this